README.md

```diff
@@ -313,6 +313,8 @@ You can also bind keys for checking the results of snitched functions and recons
   },
 ```
 
+Here's a demo of using these bindings in a shadow-cljs project: https://youtu.be/jb1BcYpyOAs
```

Comment:
I added the link to this here so I think we can remove it.
https://github.com/AbhinavOmprakash/snitch/blob/7f49580b292eb67be62890b65524030cbee4fe15/README.md?plain=1#L252-L253
```suggestion
```